base/replacer/replacer.Deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Backend for replace operations.
+  labels:
+    deploy: sourcegraph
+  name: replacer
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: replacer
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: replacer
+  template:
+    metadata:
+      labels:
+        app: replacer
+    spec:
+      containers:
+      - env:
+        - name: REPLACER_CACHE_SIZE_MB
+          value: "100000"
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: CACHE_DIR
+          value: /mnt/cache/$(POD_NAME)
+        image: index.docker.io/sourcegraph/replacer:insiders@sha256:7c376f1983652dbb899ccb72709c01bb68bcefdad488cf3bae153b2d3f3bb9f7
```

Comment:
Replacer has been pushed with an `insiders` tag. I got the hash above by running: `docker inspect --format='{{.RepoDigests}}' sourcegraph/replacer:insiders`.

Is this fine?

Reply:
Yes, that's exactly what Renovate is doing.